### ISSUE TITLE
3D Demo: New warning in the Parameterization_plugin.cpp

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#2683]
I do not understand why, but a new warning popped up last night:
```
[100%] Building CXX object Plugins/Surface_mesh/CMakeFiles/parameterization_plugin.dir/Parameterization_plugin.cpp.obj
	cd C:\CGAL\test\CGAL-4.12-Ic-117\cmake\platforms\x64_Cygwin-Windows10_MSVC2013-Debug-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh
	C:\PROGRA~2\MICROS~2.0\VC\bin\amd64\cl.exe @C:\Users\afabri\AppData\Local\Temp\nm9374.tmp
Parameterization_plugin.cpp
C:\CGAL\test\CGAL-4.12-Ic-117\cmake\platforms\x64_Cygwin-Windows10_MSVC2013-Debug-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh\Parameterization_plugin.cpp(320) : warning C4267: 'return' : conversion from 'size_t' to 'int', possible loss of data
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-117/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows10_MSVC2013-Debug-64bits.gz

Could Windows user (@janetournois @afabri) reproduce and fix that warning?

**Comment by janetournois** [comment]
I'm having a look at it

**Comment by janetournois** [comment]
I can't reproduce it in `master`
Do you have a clue about which PR introduced the warning?

However, the line
```
  int number_of_components()const{return components->size();}
```
in `Parameterization_plugin.cpp` should trigger a warning, so I submitted the PR #2684

**Comment by lrineau** [comment]
There is now another warning:
```
[ 89%] Building CXX object Plugins/Surface_mesh/CMakeFiles/parameterization_sm_plugin.dir/Parameterization_plugin.cpp.obj
	cd C:\CGAL\test\CGAL-4.12-Ic-121\cmake\platforms\x64_Cygwin-Windows10_MSVC2017-Release-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh
	C:\PROGRA~2\MICROS~1\2017\PROFES~1\VC\Tools\MSVC\1412~1.258\bin\Hostx64\x64\cl.exe @C:\Users\afabri\AppData\Local\Temp\nm47BD.tmp
Parameterization_plugin.cpp
C:\CGAL\test\CGAL-4.12-Ic-121\cmake\platforms\x64_Cygwin-Windows10_MSVC2017-Release-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh\Parameterization_plugin.cpp(751): warning C4267: 'argument': conversion from 'size_t' to 'const unsigned int', possible loss of data

```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-121/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows10_MSVC2017-Release-64bits.gz

It is in several Windows columns: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-121.shtml#Polyhedron_Demo

**Comment by MaelRL** [comment]
Should be fixed with fdd3af3.
